backend/lib/endpoints/users.js

```diff
@@ -250,7 +252,11 @@ async function routes(app) {
         authId: req.user.sub,
         email,
       };
-      return new User(userData).save();
+      const user = await new User(userData).save();
+      return {
+        ...user.toObject(),
+        organisations: [],
```

Comment:
why don't we save it as a default, `organisations: []`, instead of hard coding it

Reply:
`organisations` is a virtual based on a ref so this won't work. Hardcoding is less than ideal but this is what this #840 is about. This fix should address this in a consistent manner but let's work with this for now.